x/tally/keeper/filter.go

```diff
@@ -15,12 +16,29 @@ const (
 // ApplyFilter processes filter of the type specified in the first
 // byte of consensus filter. It returns an outlier list, which is
 // a boolean list where true at index i means that the reveal at
-// index i is an outlier, consensus boolean, and error.
+// index i is an outlier, consensus boolean, and error. It assumes
+// that the reveals and their proxy public keys are sorted.
 func ApplyFilter(input []byte, reveals []types.RevealBody) ([]int, bool, error) {
 	if len(input) == 0 {
 		return make([]int, len(reveals)), false, types.ErrInvalidFilterType
 	}
 
+	// Determine basic consensus on tuple of (exit_code, proxy_pub_keys)
+	var maxFreq int
+	freq := make(map[string]int, len(reveals))
+	for _, reveal := range reveals {
+		var success bool
+		if reveal.ExitCode == 0 {
+			success = true
+		}
```

Comment:
Can you do `success := reveal.ExitCode == 0` in Golang or is considered not done?

Reply:
Good points. That works but now Im thinking whether we should just use the exit code value rather than success boolean here. Wdyt?

Reply:
I think the result will be the same so maybe that makes more sense... Although thinking about it maybe we should do the true/false thing as we also want to be able to short circuit for the None filter?